cli/main.c

```diff
@@ -476,11 +476,13 @@ char * link_rate_conversion(uint8_t link_rate) {
 static int ltssm_log(int argc, char **argv) {
 	int ret;
 	int port;
-        int log_count, i = 0;
+	int log_count, i = 0;
```

Comment:
Don't change code you've added in an earlier patch. Squash this with that patch.